runtime-jvm/main/src/main/scala/util/Bytes.scala

```diff
@@ -66,21 +66,21 @@ object Bytes {
       (0 until size).foldLeft(Vector.empty[Byte])((buf,i) => buf :+ apply(i))
 
     /** Lexicographical minimum, assuming unsigned bytes. */
-    def min(b2: Seq): Seq = {
+    def min(b2: Seq): Seq = try {
       val sdi = this.smallestDifferingIndex(b2)
       if (sdi >= this.size) this
       else if (sdi >= b2.size) b2
       else if (unsigned(this(sdi)) < unsigned(b2(sdi))) this
       else b2
-    }
+    } catch { case Bytes.Seq.NotFound() => this }
```

Comment:
If the sequences are equal, I'm returning the first one rather than rethrowing `NotFound`.

Reply:
Seems reasonable.